src/sentry/models/grouphistory.py

```diff
@@ -24,13 +24,16 @@
 class GroupHistoryStatus:
     # Note that we don't record the initial group creation unresolved here to save on creating a row
     # for every group.
+
+    # UNRESOLVED should be used for ONGOING groups.
```

Comment:
I'm not too sure about this - we could also deprecate UNRESOLVED and add a new value for ONGOING. Not sure if that deprecation would affect any metrics.

On the other hand, redefining what a field means isn't great either

Reply:
I think defining `ONGOING = UNRESOLVED` might make sense. And then add a deprecation note to future devs to not use `UNRESOLVED` in favor of `ONGOING`

Then we can look to remove `UNRESOLVED` entirely?